src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -2,18 +2,95 @@
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final int DEFAULT_SIZE = 0;
+    private static final double CONDITION_RESIZE = 0.75;
+    private int sizeMap = DEFAULT_SIZE;
```

Comment:
```suggestion
    private static final double CONDITION_RESIZE = 0.75;
    private int size = 0;
```
розмір за замовчуванням можна не вказувати так як це і так читабельно

Reply:
+